src/common/EvalTypes.ts

```diff
@@ -37,7 +37,7 @@ export type EvaluateFnReturnType<T extends EvaluateFn> = T extends (
 /**
  * @public
  */
-export type EvaluateHandleFn = string | ((...args: unknown[]) => unknown);
+export type EvaluateHandleFn = string | Function;
```

Comment:
it's a shame that we have to make this more generic; what happens without this change? We might be able to `// @ts-expect-error` elsewhere for now (whilst this is experimental)

Reply:
(or we can use `as ...` if needed)

Reply:
Fair point. I'll revert to `((...args: unknown[]) => unknown)`